mobile/src/navigation/course-module/course-module.navigation.tsx

```diff
@@ -31,9 +27,7 @@ const CourseModule: FC = () => {
   return (
     <Tab.Navigator screenOptions={SCREEN_OPTIONS}>
       <Tab.Screen name={CourseModuleScreenName.ABOUT} component={About} />
-      {!isMentor && (
-        <Tab.Screen name={CourseModuleScreenName.TASK} component={Task} />
-      )}
+      <Tab.Screen name={CourseModuleScreenName.TASK} component={Task} />
```

Comment:
Mentor can access this screen only if he comes from my students screen, if he selects a course and goes to it's module, he shouldn't see task tab